Transformer/Model.swift

```diff
@@ -101,45 +103,45 @@ struct Attention: Layer {
         self.causal = causal
     }
     @differentiable(wrt: (self, input))
-    func applied(to input: AttentionInput, in context: Context)
+    func applied(to input: AttentionInput)
         -> Tensor<Float> {
         var dotProducts = batchedMatmul(input.query, input.key, adjointRight: true)
         dotProducts = causallyMasked(dotProducts, enable: causal) / scale
-        return batchedMatmul(dropout.applied(to: softmax(dotProducts), in: context), input.value)
+        return batchedMatmul(dropout.applied(to: softmax(dotProducts)), input.value)
     }
-    func applied(to input: AttentionInput, state: inout AttentionContext, in context: Context)
+    func applied(to input: AttentionInput, state: inout AttentionContext)
         -> Tensor<Float> {
         state = AttentionContext(
             key: state.key.concatenated(with: input.key, alongAxis: 1),
             value: state.value.concatenated(with: input.value, alongAxis: 1))
         var dotProducts = batchedMatmul(input.query, state.key, adjointRight: true)
         dotProducts = causallyMasked(dotProducts, enable: causal) / scale
-        return batchedMatmul(dropout.applied(to: softmax(dotProducts), in: context), state.value)
+        return batchedMatmul(dropout.applied(to: softmax(dotProducts)), state.value)
     }
 }
 
 @differentiable(wrt: input)
-func splitHeads(_ input: Tensor<Float>, headCount: Int32) -> Tensor<Float> {
+func splitHeads(_ input: Tensor<Float>, headCount: Int) -> Tensor<Float> {
     let (batchSize, timeSteps, features) = (input.shape[0], input.shape[1], input.shape[2])
     let featuresPerHead = features / headCount
-    let splitLastDim = input.reshaped(toShape: Tensor<Int32>(
-        [batchSize, timeSteps, headCount, featuresPerHead]))
+    let splitLastDim = input.reshaped(
+        toShape: Tensor([Int32(batchSize), Int32(timeSteps), Int32(headCount), Int32(featuresPerHead)]))
```

Comment:
```suggestion
        to: [batchSize, timeSteps, headCount, featuresPerHead])
```
Also, you can move this to the end of the previous line since it's short enough.